aiida_vasp/data/potcar.py

```diff
@@ -134,6 +133,39 @@
 from aiida_vasp.utils.aiida_utils import get_current_user, querybuild
 from aiida_vasp.utils.delegates import delegate_method_kwargs
 
+# Records for the group type strings old and new
+POTCAR_FAMILY_TYPE = 'vasp.potcar'
+OLD_POTCAR_FAMILY_TYPE = 'data.vasp.potcar.family'
+
+
+class PotcarGroup(Group):
+    """
+    A group for holding PotcarData objects that maps to various collections of POTCARs.
+    """
+
+
+def migrate_potcar_group():
```

Comment:
Great that you added the migration as well. I guess this needs to be tested. Do you want to write this as well?

Reply:
OK test added. I have also added an additional check to let the user know if migration should be performed in the `get_potcat_dict` method.

Reply:
> I have also added an additional check to let the user know if migration should be performed in the get_potcat_dict method.

@zhubonan, I got exit code 702 before migration, but (probaby) I got no message. Do you mean here I should get some message?